internal/pkg/build/sources/conveyorPacker_oci.go

```diff
@@ -211,7 +210,9 @@ func (cp *OCIConveyorPacker) Pack(ctx context.Context) (*sytypes.Bundle, error)
 }
 
 func (cp *OCIConveyorPacker) getConfig(ctx context.Context) (imgspecv1.ImageConfig, error) {
-	img, err := cp.srcRef.NewImage(ctx, cp.sysCtx)
+	// TODO - replace with ggcr code
+	//nolint:staticcheck
+	img, err := cp.srcRef.NewImage(ctx, ocitransport.SystemContextFromTransportOptions(cp.transportOptions))
```

Comment:
As above, I assume these are remnants of the `containers/image` API that can't be excised until the next major version (at the earliest)...?

Reply:
Not quite... this will be rewritten into ggcr code within later PRs for 4.1 without any change to the public API / behaviour. I didn't want to make this PR much larger by doing that in one step.